vllm/model_executor/layers/linear.py

```diff
@@ -134,6 +134,37 @@ def apply(self,
         return F.linear(x, weight, bias)
 
 
+class WeightTieLinearMethod(UnquantizedLinearMethod):
+    """Linear method base with noop create_weights
+
+    Can be used to prevent the initialization of weights
+    during the initialization of modules with weight tying.
+
+    """
+
+    def create_weights(self, layer: torch.nn.Module,
+                       input_size_per_partition: int,
+                       output_partition_sizes: List[int], input_size: int,
+                       output_size: int, params_dtype: torch.dtype,
+                       **extra_weight_attrs):
+        ...
+
+
+class QuantizationConfigOverride(QuantizationConfig):
+    """Config class for NoOPLinearMethod.
```

Comment:
> `NoOPLinearMethod`

This is not an actual type right?

Reply:
Oh, thanks! That was from an initial iteration.